style.css

```diff
@@ -0,0 +1,712 @@
+  transition: all 0.5s ease-out;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+  background-color: #fff;
+  position: absolute;
+}
+
+.container2 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 24px;
+  margin-bottom: 50px;
+  gap: 24px;
+  background-color: #fff;
+  position: absolute;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding: 12px 24px 0 24px;
+}
+
+.skills {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.skills h2 {
+  color: #000;
+  list-style-type: none;
+  font-weight: 500;
+  font-size: 20px;
+  line-height: 24px;
+}
+
+.skills img {
+  display: flex;
+  align-items: center;
+  width: 24px;
+  height: 24px;
+}
+
+.langList {
+  list-style: none;
+  padding: 0;
+}
+
+.lang {
+  background: #f7f7f9;
+  display: flex;
+  padding: 12px 9px;
+  align-items: center;
+  gap: 20px;
+  margin-bottom: 12px;
+  border-radius: 8px;
+}
+
+.skill-list li {
+  list-style: none;
+}
+
+.contact-me {
+  background-color: #6070ff;
+  background-image: url('images/contact-img-bg.png');
+  background-repeat: no-repeat;
+  background-position: right top;
+  background-size: contain;
+  border-top-left-radius: 100px;
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+  gap: 20px;
+  padding: 114px 24px 8px 24px;
+  z-index: 1;
+  margin-top: 1250px;
+}
+
+.contact-me h2 {
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  color: #fff;
+}
+
+.contact-me p {
+  font-weight: 400;
+  font-size: 20px;
+  line-height: 28px;
+  color: #ebebffff;
+}
+
+form {
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+}
+
+form input {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  color: #172b4d;
+  border-radius: 8px;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 20px;
+  padding: 15px 38px 15px 16px;
+}
+
+form textarea {
+  background: #fff;
+  border: 1px solid #cfd8dc;
+  border-radius: 8px;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #b3bac5;
+  padding: 15px 38px 15px 16px;
+}
+
+form button {
+  width: 140px;
+  height: 48px;
+  color: #6070ff;
+  background: #fff;
+  border: none;
+  letter-spacing: 0.03em;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  border-radius: 8px;
+}
+
+form button:hover {
+  background: #661d79;
+  box-shadow: 1px 8px 16px rgba(64, 83, 252, 0.24);
+  color: #fff;
+  border: none;
+  cursor: pointer;
+  transition: all 0.5s ease-out;
+}
+
+.contact-hr {
+  background: #0d151e;
+  border-radius: 10px;
+  width: 134px;
+  height: 5px;
+}
+
+.sec-box {
+  display: flex;
+  flex-direction: column;
+}
+
+.top-title {
+  display: flex;
+  justify-content: space-between;
+}
+
+.popup-image {
+  max-width: 100%;
+}
+
+.button-image {
+  width: 20px;
+}
+
+@media (min-width: 768px) {
+  body {
+    padding: 24px;
+  }
+
+  .top {
+    border-bottom-left-radius: 100px;
+  }
```

Comment:
- [ ] Kindly ensure that this container as the same width with the body element as we have it in the figma template. 👍 

> Kindly take note to implement this for `About section` and `Contact section` as well.